robottelo/constants/__init__.py

```diff
@@ -1539,8 +1539,6 @@ class Colored(Box):
     'mail': 'mail',
 }
 
-OSCAP_TARGET_CORES = 4
```

Comment:
@Gauravtalreja1 please ensure we have no more scenarios where these constants are being utilized.

Reply:
yes, all 2 tests using this constants are been replaced to use deploy_flavor, and one more test you're adding in #11813 is also using deploy_flavor which I've asked .